clang/test/Preprocessor/macro_rounding_mode.c

```diff
@@ -0,0 +1,55 @@
+// RUN: %clang_cc1 -emit-llvm -triple i386-linux -Wno-unknown-pragmas %s -o - | FileCheck %s
```

Comment:
Since this is a preprocessor testcase, can you just use -E?

Reply:
No, this macro is managed by the code in Sema, because pragma FENV_ROUND is processed there. If only `-E` is specified, __ROUNDING_MODE__ is always an empty string. We handle __FLT_EVAL_METHOD__ similarly.

Reply:
Is there some reason the preprocessor can't parse FENV_ROUND?  Breaking code with -save-temps etc. seems bad.

Reply:
Parsing pragma FENV_ROUND has been moved to the preprocessor.